src/fstapp_server.erl

```diff
@@ -1,32 +1,42 @@
 -module(fstapp_server).
 -behaviour(gen_server).
 
--export([start_link/0]).
+-export([start_link/0, stop_get_metr/0]).
```

Comment:
If we want to make this `stop_metrics` functionality "public" the common practice in Erlang is having one module as "public API" where we export all the functions available for the rest of the world.
What I mean here is we need the module `fstapp` implement a call to stop the collecting metrics, and that function will call this `fstapp_server:stop_get_metr/0`